### PR TITLE
Added notice buttons based on tags and added demo folder

### PR DESCRIPTION
**Pull request by dqdquint**
Tested with shop no: 31866 and set targetTag config to gs24. 
Example item with gs24 is "product with two tags"
Example item without gs24 is "1776 Flag Full Zip Hoodie"

### CODE REVIEW

**Review by aaron-martin — CHANGES_REQUESTED** [review]
I think it makes sense to support having more than one triggering tag. I think it makes sense to also support triggering product properties. I think it makes sense to support actions other than just linking to a phone call.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,59 @@
...
+```json
+{
+  "barColor": "orange"
+}
+```
+
+barFontColor
+```json
+{
+  "barFontColor": "whit"
```
Comment (Carsten89) [inline README.md]:
```suggestion
  "barFontColor": "white"
```

File: demo/index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+#### Merchant Admin Config:
+```json
+{
+  "triggerTags": ["is-firearm", "is-knife"],
+  "triggerProperties": [{"label": "type", "value": "firearm"}],
+  "barText": "Firearm purchases are disabled. Click to contact for purchasing.",
+  "barColor": "orange",
+  "barFontColor": "white",
+  "contactNumber": "123-456-7890",
```
Comment (Carsten89) [inline demo/index.md]: outdated. --> purchaseRedirectHref

File: README.md [hunk 3]
```diff
@@ -1,7 +1,59 @@
...
+}
+```
+
+contactNumber
```
Comment (Carsten89) [inline README.md:43]: outdated --> purchaseRedirectHref

File: README.md [hunk 4]
```diff
@@ -1,7 +1,59 @@
-# Shopgate Connect - Redirect Non-App Item Purchases
+# Shopgate Connect - Redirect Non-App Item Purchase
 
 Will prevent items with a given tag from being added to cart. A notice to call merchant will be displayed.
```
Comment (Carsten89) [inline README.md]: ..tag or property...

**Review by Carsten89 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aaron-martin) [reply README.md]: done

File: demo/index.md (hunk 2 above)
Comment (aaron-martin) [reply demo/index.md]: done

File: README.md (hunk 3 above)
Comment (aaron-martin) [reply README.md:43]: done

File: README.md (hunk 4 above)
Comment (aaron-martin) [reply README.md]: done

Comment (aaron-martin) [reply README.md]: done

**Review by Carsten89 — APPROVED** [review]

**Pull request merged**